File: src/Model/Entities/TreballadorFaGuardia.java

package hospiolot.model.business.entities;


public class TreballadorFaGuardia {

    private Guardia g;
    private Treballador t;
    
    public TreballadorFaGuardia(Guardia g, Treballador t) {
        this.g = g;
        this.t = t;
    }
    
    public Guardia getG() {
        return g;
    }

    public void setG(Guardia g) {
        this.g = g;
    }

    public Treballador getT() {
        return t;
    }

    public void setT(Treballador t) {
        this.t = t;
    }
    
    
    
}

File: BackEnd/src/Model/Entities/TreballadorFaGuardia.ts
import Guardia from "./Guardia";
import Treballador from "./Treballador";

class TreballadorFaGuardia {

    private guardia: Guardia;
    private treballador: Treballador;

    public constructor(guardia: Guardia, treballador: Treballador) {
        this.guardia = guardia;
        this.treballador = treballador;
    }

    public getG(): Guardia {
        return this.guardia;
    }

    public setG(guardia: Guardia): void {
        this.guardia = guardia;
    }

    public getT(): Treballador {
        return this.treballador;
    }

    public setT(treballador: Treballador): void {
        this.treballador = treballador;
    }
}

export = TreballadorFaGuardia;